docs/BUILD

```diff
@@ -69,3 +69,33 @@ stardoc(
         ":cc_fuzzing_rules",
     ],
 )
+
+bzl_library(
+    name = "java_fuzzing_rules",
+    srcs = [
+        "//fuzzing:instrum_opts.bzl",
+        "//fuzzing:java_defs.bzl",
+        "//fuzzing/private:binary.bzl",
+        "//fuzzing/private:common.bzl",
+        "//fuzzing/private:engine.bzl",
+        "//fuzzing/private:fuzz_test.bzl",
+        "//fuzzing/private:instrum_opts.bzl",
+        "//fuzzing/private:java_utils.bzl",
+        "//fuzzing/private:regression.bzl",
+        "//fuzzing/private/oss_fuzz:package.bzl",
+        "@rules_fuzzing_oss_fuzz//:instrum.bzl",
```

Comment:
Should we just merge `java_fuzzing_rules` and `cc_fuzzing_rules` together? We can then list them in both `cc_fuzzing_docs` and `java_fuzzing_docs`.